e2e/tools/validator/src/validator/stresser/__init__.py

```diff
@@ -65,13 +63,15 @@ def copy(self, script_path, target_script):
         self.ssh_client.exec_command(f"chmod +x {target_script}")
         logger.info("copying script %s to remote - %s - successful", script_path, target_script)
 
-    def run_script(self, script_path: str) -> ScriptResult:
+    def run_script(
+        self, script_path: str, total_runtime_seconds: int = 1200, curve_type: str = "default"
+    ) -> ScriptResult:
```

Comment:
IMHO, we should create a different function like `run_stressor` which takes specifics arguements and calls **generic** run_script function 
E.g. 
```
remote.run_stressor() -> remote.run_script(script, *args)
```

Reply:
updated